src/layouts/hass-tabs-subpage.ts

```diff
@@ -0,0 +1,225 @@
+import {
+  LitElement,
+  property,
+  TemplateResult,
+  html,
+  customElement,
+  css,
+  CSSResult,
+  PropertyValues,
+} from "lit-element";
+import "../components/ha-menu-button";
+import "../components/ha-paper-icon-button-arrow-prev";
+import { classMap } from "lit-html/directives/class-map";
+import { Route, HomeAssistant } from "../types";
+import { navigate } from "../common/navigate";
+import "@material/mwc-ripple";
+import { ConfigPageNavigation } from "../panels/config/dashboard/ha-config-navigation";
+import { isComponentLoaded } from "../common/config/is_component_loaded";
+
+@customElement("hass-tabs-subpage")
+class HassTabsSubpage extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property({ type: Boolean }) public showBackButton = true;
+  @property({ type: String, attribute: "back-path" }) public backPath?: string;
+  @property() public backCallback?: () => void;
+  @property({ type: Boolean }) public hassio = false;
+  @property({ type: Boolean }) public showAdvanced = false;
```

Comment:
Please add docstrings to each property. It's not clear to me what this property does.